Week3/Week3-JS-I.js

```diff
@@ -28,131 +28,173 @@ const newModulo = 21 % 5 === null;
 // 7
 function returnString(str) {
   //simply return the string provided: str
+  return str;
 }
 
 // 8
 function add(x, y) {
   // x and y are numbers
   // add x and y together and return the value
   // code here
+  return x + y;
 }
 
 // 9
 function subtract(x, y) {
   // subtract y from x and return the value
   // code here
+  return x - y;
 }
 
 // 10
 function multiply(x, y) {
   // multiply x by y and return the value
   // code here
+  return x * y;
 }
 
 // 11
 function divide(x, y) {
   // divide x by y and return the value
   // code here
+  return x / y;
 }
 
 // 12
 function areEqual(x, y) {
   // return true if x and y are the same
   // otherwise return false
   // code here
+  if (x === y) {
+    return true
```

Comment:
Don't forget the `;` after the return statements. Other than that this is all good! Great job!